crates/fluvio-cli/src/client/topic/create.rs

```diff
@@ -9,6 +9,7 @@ use std::io::ErrorKind;
 use std::path::PathBuf;
 use std::time::Duration;
 
+use fluvio_controlplane_metadata::topic::config::TopicConfig;
```

Comment:
Minor nit. can you move after third-party dep?

Reply:
Done